javascript/config/properties.json

```diff
@@ -0,0 +1,20 @@
+{
+  "methodProperties": [
+    {
+      "methodName": "isToggleDisabled",
+      "flagType": "control",
+      "argumentIndex": 0
+    },
+    {
+      "methodName": "isFlagTreated",
+      "flagType": "treated",
+      "argumentIndex": 0
+    },
+    {
+      "methodName": "isToggleEnabled",
+      "flagType": "treated",
+      "argumentIndex": 0
+    }
```

Comment:
let us include the `empty` flagType also.

Reply:
Is it possible to shelve this for the (not too distant) future since in order to implement deep cleaning we will need to modify all functions in `src/refactor.js` and write new tests as well? Alternatively, for this PR we can do a simple replacement `flagAPI(flag) -> null` in `flagAPIToLiteral()` and avoid deep cleaning.

Reply:
Let us shelve it. No problem.